host/integration_test.go

```diff
@@ -3722,7 +3722,7 @@ func (s *integrationSuite) TestCancelTimer() {
 			return nil, []*types.Decision{{
 				DecisionType: types.DecisionTypeStartTimer.Ptr(),
 				StartTimerDecisionAttributes: &types.StartTimerDecisionAttributes{
-					TimerID:                   common.StringPtr(fmt.Sprintf("%v", TimerID)),
+					TimerID:                   fmt.Sprintf("%v", TimerID),
```

Comment:
could probably change timerid on 3711 to a string.  but also fine to leave it 👍

Reply:
Fixed it.